src/plugins/discover/public/plugin.tsx

```diff
@@ -188,13 +188,19 @@ export class DiscoverPlugin
       order: 10,
       component: DocViewTable,
     });
-    this.docViewsRegistry.addDocView({
+    this.docViewsRegistry!.addDocView({
```

Comment:
nit: `!` seems to be a leftover of a previous commit

Reply:
good catch!